libs/hooks/open-telemetry/src/lib/otel-hook.ts

```diff
@@ -1,4 +1,4 @@
-import { FlagMetadata, Logger } from '@openfeature/server-sdk';
+import type { FlagMetadata, Logger } from '@openfeature/server-sdk';
```

Comment:
@beeme1mr I will add [this rule](https://github.com/open-feature/js-sdk/blob/f6bc6951a32fbbed9b35c1bdfd023b02874b87a1/.eslintrc.json#L15-L22) in a separate PR, which will impact probably everything in this repo, so we can't do it in this PR.